### ISSUE TITLE
core-js 3.x

### ISSUE THREAD

**Issue by blondie63** [swimlane/ngx-datatable#1694]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

* **Table version:** 
14.0.0

* **Angular version:** 
7.2.15

Is there a plan to support core-js 3.1.2 ?
At now @swimlane/ngx-datatable@14.0.0 requires a peer of core-js@^2.5.0

**Comment by FrancescoBorzi** [comment]
Confirmed. This also prevents me to update to Angular 8 using the CLI.

**Comment by marjan-georgiev** [comment]
v15.0.0 will remove core-js as a peer dependency:

https://github.com/swimlane/ngx-datatable/pull/1697

Would appreciate someone giving that PR a test run in their app.

**Comment by marjan-georgiev** [comment]
Released 15.0.0. Please verify.

**Comment by bleuscyther** [comment]
**15.0.0** Working great. No issues so far
